core/src/main/scala/com/pingcap/tispark/TiBatchWrite.scala

```diff
@@ -760,16 +899,27 @@ class ToBeCheckedRow(val row: TiRow,
                      val indexKeys: List[KeyWithDupInfo])
     extends Serializable {
 
-  //  val idxId: Long
-  // if handle is conflict or any key in indices key conflict with each other
-  // then two row equals
+  def isDuplicated: Boolean = {
+    handleKey.oldHandle != -1 || indexKeys.exists(key => key.oldHandle != -1)
+  }
+
+  // When the following case happens, two ToBeCheckedRows are considered as same:
```

Comment:
`all of the following case happens` or `one of the following case happens`?

Reply:
one of.